templates/taxi/car_list.html

```diff
@@ -1,16 +1,33 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a style="float: right" href="{% url "taxi:car-create" %}"
```

Comment:
The `<a>` tag for 'Create new one' is not properly closed. Ensure that the tag is closed correctly: `<a style="float: right" href="{% url 'taxi:car-create' %}">Create new one</a>`.